crates/core/component/stake/src/identity_key.rs

```diff
@@ -18,11 +21,83 @@ use decaf377_rdsa::{SpendAuth, VerificationKey};
 ///
 /// Using a [`SpendAuth`] key means that validators can reuse code and processes
 /// designed for custodying funds to protect their identity.
-#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(try_from = "pb::IdentityKey", into = "pb::IdentityKey")]
-pub struct IdentityKey(pub VerificationKey<SpendAuth>);
+pub struct IdentityKey {
+    /// The *compressed* bytes of the identity verification key.
+    bytes: VerificationKeyBytes<SpendAuth>,
+    // The *decompressed* identity verification key.
+    key: OnceLock<DecompressionResult>,
+}
```

Comment:
This should just be a wrapper around the `VerificationKeyBytes`:
```rust
pub struct IdentityKey(pub VerificationKeyBytes<SpendAuth>)
```
All we want to be doing here is replacing `VerificationKey` with `VerificationKeyBytes`, nothing more or less.